lib/boxen/preflight/creds.rb

```diff
@@ -80,7 +80,9 @@ def run
     unless ok?
       puts
       abort "Something went terribly wrong.",
-        "I was able to get your OAuth token, but was unable to use it."
+        "I was able to get your OAuth token, but was unable to use it." \
+	"This could be because you are using two-factor authentication or " \
+	" maybe you have an old personal access token in Github settings."
```

Comment:
It might also be worth while adding a link in here to the URL where you can view the tokens.